esp/esp/program/modules/handlers/resourcemodule.py

```diff
@@ -371,7 +371,7 @@ def resources(self, request, tl, one, two, module, extra, prog):
         if 'timeslot_form' not in context:
             context['timeslot_form'] = TimeslotForm()
         
-        context['resource_types'] = self.program.getResourceTypes().exclude(priority_default=0).order_by('priority_default')
+        context['resource_types'] = self.program.getResourceTypes().exclude(priority_default=0)
```

Comment:
Oh, whoops, I meant to get rid of the extra .exclude() in that last commit. It's not actually doing anything useful.